src/main/java/org/jabref/gui/fieldeditors/IdentifierEditor.java

```diff
@@ -25,10 +22,14 @@
 
 public class IdentifierEditor extends HBox implements FieldEditorFX {
 
-    @FXML private IdentifierEditorViewModel viewModel;
-    @FXML private EditorTextArea textArea;
-    @FXML private Button fetchInformationByIdentifierButton;
-    @FXML private Button lookupIdentifierButton;
+    @FXML
```

Comment:
Our convention is that these `@FXML` tags should be kept on the same line (I think this is also configured in the intellj style)

Reply:
It's what IntelliJ did, therefore I thought it might be overlooked in the first place. I can change this

Reply:
This is a fetish of @tobiasdiez :P For the sake of being consistent to what we have done before, please keep the `@FXML` in the same line a the instance variable.

Reply:
Found it. We need to set this in our code style for IDEA

![img](http://i.imgur.com/Z9KguGv.png)